.github/workflows/ci.yaml

```diff
@@ -24,6 +24,11 @@ jobs:
       - run: rm -rf node_modules
       - run: npm install
       - run: npm test
+      - name: coverage
```

Comment:
I am surprised this is appearing, since we removed it from the default templates.  Is this using OwlBot?

Reply:
I think it was an older docker image! I updated it.